kalkulator/src/main/java/no/nav/folketrygdloven/kalkulator/guitjenester/fakta/FastsettGrunnlagGenerell.java

```diff
@@ -28,6 +29,16 @@ public class FastsettGrunnlagGenerell {
     }
 
     public boolean skalGrunnlagFastsettes(BeregningsgrunnlagGUIInput input, no.nav.folketrygdloven.kalkulator.modell.beregningsgrunnlag.BeregningsgrunnlagPrStatusOgAndelDto andel) {
+		// Andel er blir manuelt fastsatt før, må kunne endres igjen
+		if (andel.getOverstyrtPrÅr() != null) {
```

Comment:
Kvifor må vi sjekke dette?

Reply:
For å la saksbehandler kunne fastsette andeler som allerede er blitt fastsatt. Er det ikke dumt å la de endre det en gang, og så nekte de? Eller burde de heller bruke overstyring om det skulle ha blitt feil?

Reply:
Tenkjer du for at det ikkje skal endre seg når man revurderer saken?

Reply:
Stemmer

Reply:
Eg tenkjer at dersom det er ein revurdering, så skal jo grunnlaget fastsettes på nytt. I kva situasjoner er det saksbehandler får lov til å legge til andeler i fakta om beregning der vi også skal avviksvurdere?